integrationTesting/alertListener.js

```diff
@@ -12,24 +12,23 @@ app.use(bodyParser.json());
 app.use(express.json()); // Used for smartThings wrapper
 
 // Handler for income XeThru POST requests
-app.post('/alert', async (req, res) => {
-  var responderPhone = req.body.To
-  var installationPhone = req.body.From;
-  //Complete the Chatbot
-  axios.post('/sms', {
-    To: installationPhone,
-    From: responderPhone,
-    Body: '4'
-  })
-  .then(function (response) {
-    console.log(response);
-  })
+app.post('/alert', async (req) => {
+    var responderPhone = req.body.To
+    var installationPhone = req.body.From;
+    //Complete the Chatbot
+    axios.post('/sms', {
+        To: installationPhone,
+        From: responderPhone,
+        Body: '4'
+    }).then(function (response) {
+        console.log(response);
+    })
 });
 
 app.get('/*', async function (req, res) {
     res.send('You reached the ODetect test alert listener');
 });
 
 
-server = app.listen(8081)
+app.listen(8081)
```

Comment:
So cool that the linter can pick up things like - "Hmm server isn't used later, let's not declare it at all"

Reply:
I know! It's going to be so useful. And once it's in place, then we can start adding stricter rules. I think that this will be really important as we get more people working in this code base. Consistency is so important for readability and code quality.